web/components/AttachmentMessage.js

```diff
@@ -1,52 +1,52 @@
 /**
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
```

Comment:
i dont think u need to update these if thats not when the file is made

Reply:
Okay that's fair, I wasn't sure 😅

Reply:
yeah i think thats how its usually done or else wed have to keep updating every file every year